telegram/ext/_baseupdateprocessor.py

```diff
@@ -165,7 +165,7 @@ class SimpleUpdateProcessor(BaseUpdateProcessor):
 
     async def do_process_update(
         self,
-        update: object,
+        update: object,  # noqa: ARG002
```

Comment:
tbh this seems like a bug in ARG002 - is the parent class defines this, ruff shouldn't complain …

Reply:
I'll open an issue on their repo and see what they think

Reply:
I'd still be okay with merging btw

Reply:
btw, did you open an issue?

Reply:
No, I didn't get the time to. Do you want to open one?

Reply:
I doug around a bit and found this discussion here: https://github.com/astral-sh/ruff/issues/10409#issuecomment-1998327059
Unfortunately the [`typing.override`](https://docs.python.org/3.12/library/typing.html#typing.override) decorator is new in Py3.12 and I wasn't able to trick `ruff` into using similar behavior by things like 

```python
try:
    from typing import override
except ImportError:
    override = lambda f: f
```
I guess I'll put it on our python version todo-list :)

Reply:
https://github.com/python-telegram-bot/python-telegram-bot/projects/8#card-93004105

Reply:
makes sense actually.